src/WebJobs.Extensions.DurableTask.CodeGeneration/WebJobs.Extensions.DurableTask.CodeGen.SourceGenerator/Generators/GeneratedCallerImplementationGenerator.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using WebJobs.Extensions.DurableTask.CodeGeneration.SourceGenerator.Models;
+
+namespace WebJobs.Extensions.DurableTask.CodeGeneration.SourceGenerator.Generators
+{
+    public abstract class GeneratedCallerImplementationGenerator : BaseGenerator
+    {
+        protected MethodDeclarationSyntax GenerateCallMethodWithRetry(DurableFunction function)
+        {
+            return GenerateCallerMethod(function, true);
```

Comment:
can we make the name of the second argument explicit in here?

Reply:
Yep.